### PR TITLE
[doc] Describing configuration in ASP.NET Core apps + add LogLevel config

### PR DESCRIPTION
**Pull request by gregkalapos**

### CODE REVIEW

File: docs/configuration.asciidoc [hunk 1]
```diff
@@ -1,19 +1,65 @@
 [[configuration]]
 == Configuration
 
-To adapt the Elastic APM agent to your needs, you can configure it using environment variables.
+To adapt the Elastic APM agent to your needs, you can configure it using different configuration sources, which have different naming conventions for the property key. 
 
+By default the agent uses environment variables. Additionally on ASP.NET Core the agent can plug into the https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-2.2[Microsoft.Extensions.Configuration] infrastructure.
```
Comment (bmorelli25) [inline docs/configuration.asciidoc]:
```suggestion
By default, the agent uses environment variables. Additionally, on ASP.NET Core, the agent can plug into the https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-2.2[Microsoft.Extensions.Configuration] infrastructure.
```

Comment (bmorelli25) [reply docs/configuration.asciidoc]: Added a couple commas ^

File: docs/configuration.asciidoc [hunk 2]
```diff
@@ -1,19 +1,65 @@
 [[configuration]]
 == Configuration
 
-To adapt the Elastic APM agent to your needs, you can configure it using environment variables.
+To adapt the Elastic APM agent to your needs, you can configure it using different configuration sources, which have different naming conventions for the property key.
```
Comment (bmorelli25) [inline docs/configuration.asciidoc]: This sentence is a bit rough. What about something like this:

```suggestion
You can utilize configuration options to adapt the Elastic APM agent to your needs. There are multiple configuration sources, each with different naming conventions for the property key.
```

File: docs/configuration.asciidoc [hunk 3]
```diff
@@ -1,19 +1,65 @@
 [[configuration]]
 == Configuration
 
-To adapt the Elastic APM agent to your needs, you can configure it using environment variables.
+To adapt the Elastic APM agent to your needs, you can configure it using different configuration sources, which have different naming conventions for the property key. 
 
+By default the agent uses environment variables. Additionally on ASP.NET Core the agent can plug into the https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-2.2[Microsoft.Extensions.Configuration] infrastructure.
+
+=== Configuration on ASP.NET Core
+The `UseElasticApm()` extension method offers an overload to pass an `IConfiguration` instance to the APM Agent. By using this overload in a typical ASP.NET Core application the `Startup.cs` file would contain code similar to the following:
```
Comment (bmorelli25) [inline docs/configuration.asciidoc]: +comma
```suggestion
The `UseElasticApm()` extension method offers an overload to pass an `IConfiguration` instance to the APM Agent. By using this overload in a typical ASP.NET Core application, the `Startup.cs` file would contain code similar to the following:
```

File: docs/configuration.asciidoc [hunk 4]
```diff
@@ -26,3 +72,23 @@ The URLs for your APM Servers. The URLs must be fully qualified, including proto
 
 
 NOTE: Providing multiple URLs only works with APM Server v6.5+.
+
+[float]
+[[config-LogLevel]]
```
Comment (bmorelli25) [inline docs/configuration.asciidoc]: Section IDs should be lowercase
```suggestion
[[config-loglevel]]
```

File: docs/configuration.asciidoc [hunk 5]
```diff
@@ -1,19 +1,65 @@
...
+    }
+}
+----
 
 [[config-reporter]]
 === Reporter configuration options
 [float]
-[[config-server-urls]]
-==== `server_urls`
+[[config-ServerUrls]]
```
Comment (bmorelli25) [inline docs/configuration.asciidoc]: section IDs should be lowercase
```suggestion
[[config-serverurls]]
```

File: docs/configuration.asciidoc (hunk 4 above)
Comment (bmorelli25) [reply docs/configuration.asciidoc]: ignore this comment

File: docs/configuration.asciidoc (hunk 5 above)
Comment (bmorelli25) [reply docs/configuration.asciidoc]: sorry, ignore above, should actually be this:
```suggestion
[[config-server-urls]]
```

File: docs/configuration.asciidoc (hunk 2 above)
Comment (gregkalapos) [reply docs/configuration.asciidoc]: This was literally copy pasted from the java doc 😀😀 But I agree, the suggestion is good.

File: docs/configuration.asciidoc [hunk 6]
```diff
@@ -1,19 +1,65 @@
...
+    }
+}
+----
 
 [[config-reporter]]
 === Reporter configuration options
 [float]
 [[config-server-urls]]
-==== `server_urls`
+==== `ServerUrls`
```
Comment (bmorelli25) [inline docs/configuration.asciidoc]: Looks like this might've been missed:
```suggestion
==== `config-server-urls`
```

**Comment by gregkalapos** [comment]
Thx @bmorelli25!

**Pull request merged**